src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -276,6 +306,8 @@ def load_arguments(self, _):
         c.argument('restore_only_osdisk', arg_type=get_three_state_flag(), help='Use this flag to restore only OS disks of a backed up VM.')
         c.argument('restore_as_unmanaged_disks', arg_type=get_three_state_flag(), help='Use this flag to specify to restore as unmanaged disks')
         c.argument('use_secondary_region', action='store_true', help='Use this flag to show recoverypoints in secondary region.')
+        c.argument('rehydration_duration', type=int, help='Maximum time, in days, the recovery point stays in hydrated state.')
```

Comment:
Specify the default value which will be used in case user does not provide this input. 15 in your case.

Reply:
So, i have specified that in the function definition...where we write the argument = None, instead of that i have put argument = 15...it does the job. I have tried not passing that hydration-duration parameter and it takes it as 15 in that case.

Reply:
Yeah. I am telling that in the help text just specify "Default: 15" just so that the user gets aware. Check --cross-region-restore-flag parameter for context.

Reply:
done